source/howtos/updating_pelican_sites.rst

```diff
@@ -0,0 +1,406 @@
+CASS Website Workflow
+=====================
+
+This documentation applies to both the CASS main website, the OSL website,
+and any other sites using the Pelican static site generator.
+
+
+What is Pelican
+---------------
+
+Pelican is a static site generator. In short, it allows us to create a 
+great website by writing simple text files and checking them into Git.
+
+
+What is Git
+-----------
+
+Git is a system that tracks changes to files. It allows multiple people
+to edit the same set of files and merge their changes together. All the
+files, and a history of all the changes made to them, are stored in such
+a way that you can retrieve the past state of any file easily. 
+
+We store our Pelican site files in Git, and we use Github to store 
+those git repositories. Github provides an easy to use web interface to
+the majority of git's functionality. We use it to review changes, to
+merge changes into production, and to track issues.
+
+
+Git via GitHub
+~~~~~~~~~~~~~~
+
+You can do most git operations entirely from the Github web UI. If 
+installing a git client on your personal workstation is problematic,
+you can create and edit site content entirely online.
+
+
+Git on the Mac
+~~~~~~~~~~~~~~
+
+
+
+Git on Windows
+~~~~~~~~~~~~~~
+
+
+
+The Workflow
+------------
+
+
+**Summary**
+
+#.	Check out the Git repository and create a new branch
+#.	Add/edit content
+#.	Push your branch up to Github
+#.	Make a pull request on Github
+#.	Solicit reviews of your changes
+#.	Review changes on the staging website
+#.	Merge staging with production
+
+
+Checking out the repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The content for Pelican sites will be in a repository on Github named
+for the site it contains:
+
+* OSL website: https://github.com/osuosl/osuosl-pelican
+* CASS website: https://github.com/osuosl/cass-pelican
+* Beaver Barcamp: https://github.com/osuosl/beaver-barcamp-pelican
+
+You can view and even edit the contents of the repository directly on its 
+Github page, but it is usually better to 'clone' the repository to your
+local machine. How you do this depends on the operating system and git 
+client you use. For examples in this document, I'll use the git 
+command-line client, which can be used on all platforms. 
+
+.. code-block:: none
```

Comment:
If you're not setting what type of code block this is, you can just use ``::`` instead. Can you please do that for all instances using ``none`` please?